llama_stack/distribution/build_container.sh

```diff
@@ -268,7 +262,7 @@ fi
 # Add other require item commands genearic to all containers
 add_to_container << EOF
 
-RUN chmod -R g+rw /app /.llama /.cache
```

Comment:
The cache is actually for `huggingface`, I'm afraid to break something if we remove it. For instance, the cache gets populated by embedding models via the sentence-transformers library.

Let's keep it, it must be here for a reason 🙏🏻